commands/user.js

```diff
@@ -0,0 +1,132 @@
+/* jshint esversion: 11 */
+
+const {
+	SlashCommandBuilder,
+	ActionRowBuilder,
+	ButtonBuilder,
+	ButtonStyle
+} = require('discord.js');
+const config = require("../config/aod-discord-bot.config.json");
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('user')
+		.setDescription('Manage a user')
+		.addSubcommand(command => command.setName('kick').setDescription('Kicks a user from the server')
+			.addUserOption(option => option.setName('user').setDescription('User').setRequired(true))
+			.addStringOption(option => option.setName('reason').setDescription('Kick reason')))
+		.addSubcommand(command => command.setName('ban').setDescription('Bans a user from the server')
+			.addUserOption(option => option.setName('user').setDescription('User').setRequired(true))
+			.addStringOption(option => option.setName('reason').setDescription('Ban reason'))
+			.addIntegerOption(option => option.setName('delete-messages').setDescription('Message purge duration')
+				.addChoices({ name: '10 minutes', value: 600 }, { name: '30 minutes', value: 1800 }, { name: '1 hour', value: 3600 }, { name: '1 day', value: 86400 }, ))),

```

Comment:
To help with code readability, can you pull the list of choices out to a separate list and use the spread operator? Should be some examples in channel.js.

I'd suggest we need a none option since some people will always try to use every parameter. The values also go up to 7 days and we might as well include that as well.

Reply:
Resolved in 7bbcdcc